core/src/main/scala/filodb.core/query/QueryConfig.scala

```diff
@@ -83,7 +83,7 @@ object QueryConfig {
 }
 case class RoutingConfig(
                           supportRemoteRawExport: Boolean                = false,
-                          maxRemoteRawExportTimeRange: FiniteDuration    = 3 days,
+                          maxRemoteRawExportTimeRange: FiniteDuration    = 31 days,
```

Comment:
Leave this default as-is (better to configure this externally because it's unclear yet what new problems the change from 3d -> 31d might uncover).

Reply:
You are right. In Splunk, I've seen lots of query blocked by this validation, so we should leave as it is.